index.js

```diff
@@ -29,20 +33,104 @@ cp.parseBlock = function () {
 	return this.finishNode(node, "BlockStatement");
 };
 
+function strToModuleIdentifier(str) {
```

Comment:
Does Node v4 support `let` though?

Also - are we assuming that source files are just ES5 but with `import`/`export` in that case?